interface/src/app/analyze/analyze.js

```diff
@@ -52,6 +53,21 @@ angular.module('adage.analyze', [
         return classList;
       },
 
+      item_tooltip: function(search_item) {
```

Comment:
Hmmm, I think I might be missing something - any reason for using underscores instead of camelCase (here and also above in the `item_style` function and the `search_item` argument, and below in the `scroll_to_id` function)?